### PR TITLE
Pulpcore 3.2 - gem updates

### PR DESCRIPTION
**Pull request by sjha4**
Supported Versions:

 * Nightly

### CODE REVIEW

File: packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec [hunk 1]
```diff
@@ -4,25 +4,21 @@
 
 %global gem_name pulp_2to3_migration_client
 
-%global release 1
-%global prerelease rc1
-%global prereleaserpm %{?prerelease:.}%{?prerelease}
-
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.0.1
-Epoch: 1
```
Comment (evgeni) [inline packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec]: please don't delete the epoch

File: packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec [hunk 2]
```diff
@@ -4,25 +4,21 @@
...
-Release: %{?prereleaserpm:0.}%{release}%{?prereleaserpm}%{?dist}
-Summary: Pulp 2 to Pulp 3 API Ruby Gem
+Version: 0.2.0b1
```
Comment (evgeni) [inline packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec]:
```suggestion
Version: 0.2.0
```
and then set `prerelease b1` above

File: packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec [hunk 3]
```diff
@@ -4,25 +4,21 @@
 
 %global gem_name pulp_2to3_migration_client
 
-%global release 1
-%global prerelease rc1
-%global prereleaserpm %{?prerelease:.}%{?prerelease}
```
Comment (evgeni) [inline packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec]: please don't delete these

File: packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec [hunk 4]
```diff
@@ -4,25 +4,21 @@
...
-Summary: Pulp 2 to Pulp 3 API Ruby Gem
+Version: 0.2.0b1
+Release: 1%{?dist}
+Summary: Pulp 3 API Ruby Gem
 Group: Development/Languages
-License: GPLv2
+License: GPL-2.0+
 URL: https://github.com/pulp/pulp-2to3-migration
-Source0: https://rubygems.org/gems/%{gem_name}-%{version}%{?prerelease}.gem
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
```
Comment (evgeni) [inline packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec]:
```suggestion
Source0: https://rubygems.org/gems/%{gem_name}-%{version}%{?prerelease}.gem
```

File: packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec [hunk 5]
```diff
@@ -4,25 +4,21 @@
...
 
 # start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby >= 1.9
 Requires: %{?scl_prefix_ruby}ruby(rubygems) > 1.3.1
 Requires: %{?scl_prefix}rubygem(faraday) >= 0.14.0
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1.0
 Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1
```
Comment (evgeni) [inline packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec]: this seems duplicated

File: packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec [hunk 6]
```diff
@@ -38,7 +34,7 @@ Provides a client for pulp2 to pulp3 migration support within pulp3
 %package doc
 Summary: Documentation for %{pkg_name}
 Group: Documentation
-Requires: %{?scl_prefix}%{pkg_name} = %{epoch}:%{version}-%{release}
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
```
Comment (evgeni) [inline packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec]:
```suggestion
Requires: %{?scl_prefix}%{pkg_name} = %{epoch}:%{version}-%{release}
```

File: packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec [hunk 7]
```diff
@@ -49,7 +45,7 @@ Documentation for %{pkg_name}.
 gem unpack %{SOURCE0}
 %{?scl:EOF}
 
-%setup -q -D -T -n  %{gem_name}-%{version}%{?prerelease}
+%setup -q -D -T -n  %{gem_name}-%{version}
```
Comment (evgeni) [inline packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec]:
```suggestion
%setup -q -D -T -n  %{gem_name}-%{version}%{?prerelease}
```

File: packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec [hunk 8]
```diff
@@ -74,9 +70,9 @@ cp -a .%{gem_dir}/* \
 
 %files
 %dir %{gem_instdir}
-%exclude %{gem_instdir}/git_push.sh
-%exclude %{gem_cache}
+%{gem_instdir}/git_push.sh
```
Comment (evgeni) [inline packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec]:
```suggestion
%exclude  %{gem_instdir}/git_push.sh
```

File: packages/katello/rubygem-pulp_ansible_client/rubygem-pulp_ansible_client.spec [hunk 9]
```diff
@@ -1,29 +1,26 @@
...
 # start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby >= 1.9
 Requires: %{?scl_prefix_ruby}ruby(rubygems) > 1.3.1
-Requires: %{?scl_prefix}rubygem(typhoeus) >= 1.0
-Requires: %{?scl_prefix}rubygem(typhoeus) < 2
-Requires: %{?scl_prefix}rubygem(typhoeus) >= 1.0.1
+Requires: %{?scl_prefix}rubygem(faraday) >= 0.14.0
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1.0
 Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1
```
Comment (evgeni) [inline packages/katello/rubygem-pulp_ansible_client/rubygem-pulp_ansible_client.spec]: duplicate?

File: packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec [hunk 10]
```diff
@@ -4,17 +4,17 @@
...
+%global prerelease b1	
 %global prereleaserpm %{?prerelease:.}%{?prerelease}
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.0.1
 Epoch: 1
-Release: %{?prereleaserpm:0.}%{release}%{?prereleaserpm}%{?dist}
-Summary: Pulp 2 to Pulp 3 API Ruby Gem
+Version: 0.2.0
+Release: 1%{?dist}
```
Comment (sjha4) [inline packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec]: @evgeni : Does this look right to you? Or should I just keep 
` %{?prereleaserpm:0.}%{release}%{?prereleaserpm}%{?dist}`

**Pull request merged**